server/service/service.go

```diff
@@ -42,6 +44,9 @@ func NewService(ds kolide.Datastore, resultStore kolide.QueryResultStore,
 		osqueryResultLogWriter: resultWriter,
 		mailService:            mailService,
 		ssoSessionStore:        sso,
+		metaDataClient: http.Client{
```

Comment:
usually http.Client is a pointer. not sure if it makes a huge difference, but `http.DefaultClient` is also an `&http.Client{}`